### DISCUSSION TITLE
Extract image headers automatically

### DISCUSSION THREAD

**Discussion by MarceloSanchez-Aerial** [mozman/ezdxf#501] [Issues]
Hi,
When using zwcad (a cad tool) i can import automatically .ecw or .tif images without needing the define , center of the image or pixel size. The program is able to extract the headers automatically, is there a way to do this in ezdxf?

![image](https://user-images.githubusercontent.com/84915678/126455408-292f9b64-b95a-49c6-b951-d225fcb7b309.png)

**Comment by mozman** [comment]
No, there is no API for that.
